### PR TITLE
Rename variable and initialize properly

### PR DESCRIPTION
**Pull request by sslaws**
The new jobs pruning portion of the script was failing as the variable was not being initialized properly.
I renamed the variable jobs as it is valid yaml syntax and I wanted to be sure we had no conflicts.

### CODE REVIEW

File: .github/workflows/prune-images.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ on:
         description: "Comma seperated list of application Image Streams to prune"
         required: true
         default: "web-sims, api-sims, queue-consumers-sims, workers-sims"
-      jobs:
+      ocjobs:
```
Comment (andrewsignori-aot) [inline .github/workflows/prune-images.yml:20]: Can it follow the same camel case pattern from `minTags` which would be `ocJobs`?

File: .github/workflows/prune-images.yml [hunk 2]
```diff
@@ -40,14 +40,15 @@ jobs:
         run: |
           echo "ENVIRONMENT=${{ inputs.environment || 'test' }}" >> $GITHUB_ENV
           echo "APPLICATIONS=${{ inputs.applications || 'web-sims, api-sims, queue-consumers-sims, workers-sims' }}" >> $GITHUB_ENV
+          echo "OCJOBS=${{ inputs.ocjobs || 'db.migrations' }}" >> $GITHUB_ENV
```
Comment (andrewsignori-aot) [inline .github/workflows/prune-images.yml:43]: Can it follow the same naming pattern from `MIN_TAGS`?

**Review by andrewsignori-aot — APPROVED** [review]
Just added minor comments. Looks good 👍

**Review by dheepak-aot — APPROVED** [review]
Looks good @sslaws 👍

**Pull request merged**